### ISSUE TITLE
Video without sound raises AttributeError

### ISSUE THREAD

**Issue by 8ba979b9-7a35-4eed-8a0e-b4dc97eea618** [pyglet/pyglet#157]
**Description:** Playing a video after one that doesn't contain sound using `player.next_source()`
raises an AttributeError.

**Pyglet Version(s):** 1.4.10, 1.5.0
**Python Version(s):** 3.8

```
"../site-packages/pyglet/media/player.py", line 333, in next_source
AttributeError: 'NoneType' object has no attribute 'delete'
```
(Pyglet 1.5.0)

`self._audio_player` (in `pyglet/media/player.py`) seems to be `None` for videos without an audio track.
This isn't checked before `self._audio_player.delete()` which causes this AttributeError.

**Comment by benmoran56** [comment]
Thanks for opening the ticket!
This should now be fixed in pyglet-1.5-maintenance branch, and will be in the next point release.

**Comment by benmoran56** [comment]
This is fixed in the latest releases. Closing.
